modules/run-vault/run-vault

```diff
@@ -137,11 +143,30 @@ function generate_vault_config {
   local readonly enable_s3_backend="$8"
   local readonly s3_bucket="$9"
   local readonly s3_bucket_region="${10}"
+  local readonly enable_auto_unseal="${11}"
+  local readonly auto_unseal_kms_key="${12}"
+  local readonly auto_unseal_kms_key_region="${13}"
   local readonly config_path="$config_dir/$VAULT_CONFIG_FILE"
 
   local instance_ip_address
   instance_ip_address=$(get_instance_ip_address)
 
+  local auto_unseal_config=""
+  if [[ "$enable_auto_unseal" == "true" ]]; then
+    auto_unseal_config=$(cat <<EOF
+seal "awskms" {
+  kms_key_id = "$auto_unseal_kms_key"
+  region = "$auto_unseal_kms_key_region"
+}\n
+EOF
+)
+  fi
+
+  # kms_key_id = "d7c1ffd9-8cce-45e7-be4a-bb38dd205966"
+  # region      = "$auto_unseal_region"
+  # access_key = "AKIAIOSFODNN7EXAMPLE"
+  # secret_key = "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
```

Comment:
Please remove this comment and if these are real keys, revoke them 😄

Reply:
these dont exist anymore, but oops :)